src/data/equiv/basic.lean

```diff
@@ -1540,6 +1540,19 @@ by { dsimp [set_value], simp [swap_apply_left] }
 
 end swap
 
+protected lemma exists_unique_congr (p : β → Prop) (e : α ≃ β) :
+  (∃! (y : β), p y) ↔ ∃! (x : α), p (e x) :=
```

Comment:
Based on the definition of `forall_congr`, I'd expect this to be
```suggestion
protected lemma exists_unique_congr {p : α → Prop} {q : β → Prop} (f : α ≃ β)
  (h : ∀{x}, p x ↔ q (f x)) : (∃! x, p x) ↔ ∃! y, q y :=
```

Reply:
Thanks!